_includes/themes/restart/default.html

```diff
@@ -85,6 +85,9 @@
           </div>
         </section>
       </footer>
+
+      {% include themes/restart/modal.html %}
```

Comment:
Including this into default.html means EVERY page gets this!
This is not OK!!!

Either include into `page-jobad-list.html` or make it part of this file

Reply:
ok